tests/cephfs/clients/verify_root_squash_in_caps.py

```diff
@@ -0,0 +1,215 @@
+import random
+import string
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    CEPH-83573868
+    Test Steps:
+    1. Create 2 CepFS and subvolume in each FS
+    2. Create client user with root_squash cap for / and regular permissions for /volumes on both CephFS and verify
+    3. Perform Kernel and Fuse mount of CephFS
+    4. Validate root_squash is working on / and write permissions work on /volumes
+    5. cleanup
+    """
+    try:
+        tc = "CEPH-83573868"
+        log.info("Running cephfs %s test case" % (tc))
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        build = config.get("build", config.get("rhbuild"))
+        clients = ceph_cluster.get_ceph_objects("client")
+        client = clients[0]
+        log.info("checking Pre-requisites")
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+
+        log.info(
+            "Create Test configuration - Additional CephFS and one subvolume in each CephFS"
+        )
+        client.exec_command(sudo=True, cmd="ceph fs volume create cephfs_1")
+        fs_list = ["cephfs", "cephfs_1"]
```

Comment:
there might be chances of cephfs not present in the cluster.Can we add validation for it 
```
fs_details = fs_util.get_fs_info(client)
        if not fs_details:
            fs_util.create_fs(client, "cephfs")
```